jackson-modules/jackson-annotations/src/test/java/com/baeldung/jackson/deduction_based_polymorphism/ContainedInferenceUnitTest.java

```diff
@@ -0,0 +1,61 @@
+package com.baeldung.jackson.deduction_based_polymorphism;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.json.JsonMapper;
+import org.junit.jupiter.api.Test;
+
+import static com.baeldung.jackson.deduction_based_polymorphism.JsonStringFormatterUtil.formatJson;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNull;
+import static org.junit.jupiter.api.Assertions.assertSame;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+public class ContainedInferenceUnitTest {
```

Comment:
Since you are using JUnit 5, you can remove all the public class modifiers, they are not necessary.